src/site/stages/build/plugins/tests/check-collections.spec.js

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+import checkCollections from '../check-collections';
+import ware from 'ware';
```

Comment:
Is `ware` installed in our project? 

Also, curious about your choice with using `ware`, versus working with the plugin directly.